modules/mobile/app/models/mobile/v0/adapters/claims_overview.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    module Adapters
+      class ClaimsOverview
+        def parse(entry)
+          if entry['list_data']
+            {
+              id: entry['evss_id'].to_s,
+              type: 'claim',
+              subtype: entry['list_data']['status_type'],
```

Comment:
when you're traversing down two nodes `.dig('list_data', 'status_type')` will protect you from hitting a nil value along the way

Reply:
Unfortunately it looks like dig doesn't work with this object.
` undefined method `dig' for #<EVSSClaim:0x00007fe00b0eed48>`